src/compute.c

```diff
@@ -12,7 +12,9 @@
 
 #include "ooura/dct.h"
 
+#ifdef _WIN32
```

Comment:
Why filter FLT_EVAL_METHOD as if it was win32-specific? This is standard C99, ref. https://en.wikipedia.org/wiki/C99#IEEE_754_floating_point_support

Reply:
compute.c: At top level:
compute.c:15:15: error: expected declaration specifiers or ‘...’ before numeric constant
compute.c:15:37: error: expected declaration specifiers or ‘...’ before string constant